pkg/entrypoint/entrypointer_test.go

```diff
@@ -93,6 +94,43 @@ func TestEntrypointerFailures(t *testing.T) {
 			if c.postFile == "" && fpw.wrote != nil {
 				t.Errorf("Wrote post file when not required")
 			}
+
+			if c.waiter == nil {
+				fileContents, err := ioutil.ReadFile("termination")
+				if err == nil {
+					var entries []v1alpha1.PipelineResourceResult
+					if err := json.Unmarshal([]byte(fileContents), &entries); err == nil {
+						index := make(map[string]v1alpha1.PipelineResourceResult, len(entries))
+						for _, item := range entries {
+							index[item.Key] = item
+						}
+						if _, ok := index["StartedAt"]; !ok {
+							t.Error("Didn't find the StartedAt entry")
+						}
+						if reason, ok := index["Reason"]; ok {
+							if reason.Value != "Unknown" {
+								t.Error("Reason should be Unknown")
+							}
+						} else {
+							t.Error("Didn't find the Reason entry")
+						}
+
+						if message, ok := index["Message"]; ok {
+							if message.Value != "runner failed" {
```

Comment:
nit: could you put the `"runner failed"` string into a shared const so that it can be used both here and in `func (f *fakeErrorRunner) Run(ctx context.Context, args ...string) error`.  This just makes it a tiny bit easier to ensure that the string remains the same in both places.